lucien-ios/AddAndEditNewComicBook/ComicFormViewController.swift

```diff
@@ -89,12 +121,12 @@ final class AddComicViewController: UIViewController, AlertDisplaying {
     }
 
     private func setNavBarTitle() {
-        navigationController?.viewControllers[0].title = "Add Book"
+        navigationController?.viewControllers[0].title = comicFormViewModel?.comicFormMode == .add ?  "Add Book" : "Edit Book"
```

Comment:
This is something that should come from the view model directly. It should expose a `navBarTitle` property (can be computed just like this) so the view controller doesn't need to think about it